coderd/database/queries/aibridge.sql

```diff
@@ -1,40 +1,160 @@
 -- name: InsertAIBridgeInterception :one
-INSERT INTO aibridge_interceptions (id, initiator_id, provider, model, metadata, started_at)
-VALUES (@id::uuid, @initiator_id::uuid, @provider, @model, COALESCE(@metadata::jsonb, '{}'::jsonb), @started_at)
+INSERT INTO aibridge_interceptions (
+	id, initiator_id, provider, model, metadata, started_at
+) VALUES (
+	@id, @initiator_id, @provider, @model, COALESCE(@metadata::jsonb, '{}'::jsonb), @started_at
+)
 RETURNING *;
 
--- name: InsertAIBridgeTokenUsage :exec
+-- name: InsertAIBridgeTokenUsage :one
 INSERT INTO aibridge_token_usages (
   id, interception_id, provider_response_id, input_tokens, output_tokens, metadata, created_at
 ) VALUES (
   @id, @interception_id, @provider_response_id, @input_tokens, @output_tokens, COALESCE(@metadata::jsonb, '{}'::jsonb), @created_at
-);
+)
+RETURNING *;
 
--- name: InsertAIBridgeUserPrompt :exec
+-- name: InsertAIBridgeUserPrompt :one
 INSERT INTO aibridge_user_prompts (
   id, interception_id, provider_response_id, prompt, metadata, created_at
 ) VALUES (
   @id, @interception_id, @provider_response_id, @prompt, COALESCE(@metadata::jsonb, '{}'::jsonb), @created_at
-);
+)
+RETURNING *;
 
--- name: InsertAIBridgeToolUsage :exec
+-- name: InsertAIBridgeToolUsage :one
 INSERT INTO aibridge_tool_usages (
   id, interception_id, provider_response_id, tool, server_url, input, injected, invocation_error, metadata, created_at
 ) VALUES (
   @id, @interception_id, @provider_response_id, @tool, @server_url, @input, @injected, @invocation_error, COALESCE(@metadata::jsonb, '{}'::jsonb), @created_at
-);
+)
+RETURNING *;
 
 -- name: GetAIBridgeInterceptionByID :one
-SELECT * FROM aibridge_interceptions WHERE id = @id::uuid;
+SELECT
+	*
+FROM
+	aibridge_interceptions
+WHERE
+	id = @id::uuid;
 
 -- name: GetAIBridgeInterceptions :many
-SELECT * FROM aibridge_interceptions;
+SELECT
+	*
+FROM
+	aibridge_interceptions;
 
 -- name: GetAIBridgeTokenUsagesByInterceptionID :many
-SELECT * FROM aibridge_token_usages WHERE interception_id = @interception_id::uuid;
+SELECT
+	*
+FROM
+	aibridge_token_usages WHERE interception_id = @interception_id::uuid
+ORDER BY
+	created_at DESC,
```

Comment:
Kinda weird that these are sorted DESC and their `List*` counterparts are reversed. Why is that?